pkg/middleware/requestmeta/request_metadata.go

```diff
@@ -13,10 +13,18 @@ const (
 	TeamCore     = "core"
 )
 
+type StatusSource string
+
+const (
+	StatusSourceServer     StatusSource = "server"
```

Comment:
No strong opinions changing this in request_metadata. However, considering leaving errutil as is since we might want to keep this generic. Asks regarding plugins/other repositories wanting to use the errutil package and the current suggestion is to use https://github.com/grafana/gerror. Still a bit up in the air whether or when errutil will go into https://github.com/grafana/gerror.

Reply:
:+1: Makes sense, let's leave it like this then